debezium-server/debezium-server-pravega/src/main/java/io/debezium/server/pravega/PravegaChangeConsumer.java

```diff
@@ -46,7 +46,7 @@
     @ConfigProperty(name = PROP_CONTROLLER, defaultValue = "tcp://localhost:9090")
     URI controllerUri;
 
-    @ConfigProperty(name = PROP_SCOPE)
+    @ConfigProperty(name = PROP_SCOPE, defaultValue = "debezium")
```

Comment:
This seems like a reasonable default.

I don't know if Debezium has precedent for default group or namespace names elsewhere already. Maybe its Schema Registry integration or...?

Reply:
I don't think there's precedent for this elsewhere, so this name is as good as any other.